lusidtools/cocoon/utilities.py

```diff
@@ -915,3 +915,11 @@ def validate_mapping_file_structure(mapping: dict, columns: list, file_type: str
         raise ValueError(f"'required' mapping field not provided in mapping")
 
     pass
+
+
+def strip_whitespace(df: pd.DataFrame) -> pd.DataFrame:
+    for col in list(df.columns):
```

Comment:
You can use df = df.apply(lambda x: x.strip() if isinstance(x, str) else x) to make this much simpler